src/cli.ts

```diff
@@ -1,33 +1,78 @@
 #!/usr/bin/env node
 
+import fs from 'fs/promises';
+import path from 'path';
+import crypto from 'crypto';
+import dotenv from 'dotenv';
 import { Command } from "commander";
-import { authenticate } from "./commands/auth";
-import { install } from "./commands/install";
-import { deploy } from "./commands/deploy";
-import figlet from "figlet";
+import { authenticate, checkAuth } from "./commands/auth.js";
+import { install } from "./commands/install.js";
+import { deploy } from "./commands/deploy.js";
+import figlet from 'figlet';
 
-console.log(figlet.textSync("Anyflow CLI"));
+async function ensureEnvFile() {
+  const envPath = path.resolve(process.cwd(), '.env');
```

Comment:
`process.cwd()` will return the current directory the cli is running on, not the project's current directory. Example of how this can be an issue:

```
cd my-project/subdirectory
anyflow init
```

the `.env` file will be stored on `my-project/subdirectory/.env` and not on the project root

Reply:
Okay solved 👍🏻